sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -289,3 +289,28 @@ $border-color-info: #6554c0;
 .min-width-modal {
   min-width: 500px;
 }
+
+.restriction-modal {
+  min-width: 600px;
```

Comment:
do we really need this?

Reply:
as discussed, i believe the modals are responsive

Reply:
TODO comment added as discussed.